package.nls.json

```diff
@@ -1,19 +1,19 @@
-    "settings.showNotifications.off.description": "All notifications are turned off, any errors or warning are still available in the logs.",
-    "settings.showNotifications.onError.description": "Notifications are shown only in the case of an error.",
-    "settings.showNotifications.onWarning.description": "Notifications are shown for errors and warnings.",
-    "settings.showNotifications.always.description": "Notifications are shown for anything that the server chooses to show."
+    "settings.args.description": "Arguments passed to Pylint for linting Python files. Each argument should be provided as a separate string in the array. \n Examples:  \n- `\"pylint.args\": [\"--rcfile=<file>\"]` \n - `\"pylint.args\": [\"--disable=C0111\", \"--max-line-length=120\"]`",
+    "settings.cwd.description": "Sets the current working directory used to lint Python files with Pylint. By default, it uses the root directory of the workspace `${workspaceFolder}`. You can set it to `${fileDirname}` to use the parent folder of the file being linted as the working directory for Pylint.",
+    "settings.severity.description": "Mapping of Pylint's message types to VS Code's diagnostic severity levels as displayed in the Problems window. You can also use it to override specific Pylint error codes. \n Example:</br> `{\"convention\": \"Information\", \"error\": \"Error\", \"fatal\": \"Error\", \"refactor\": \"Hint\", \"warning\": \"Warning\", \"W0611\": \"Error\", \"undefined-variable\": \"Warning\"}`",
+    "settings.lintOnChange.description": "Enable linting Python files with Pylint as you type.",
+    "settings.path.description": "Path or command to be used by the extension to lint Python files with Pylint. Accepts an array of a single or multiple strings. If passing a command, each argument should be provided as a separate string in the array. If set to `[\"pylint\"]`, it will use the version of Pylint available in the `PATH` environment variable. Note: Using this option may slowdown linting. \nExamples: \n- `[\"~/global_env/pylint\"]` \n- `[\"conda\", \"run\", \"-n\", \"lint_env\", \"python\", \"-m\", \"pylint\"]` \n `[\"pylint\"]`",
+    "settings.ignorePatterns.description": "Configure [glob patterns](https://code.visualstudio.com/docs/editor/glob-patterns) to exclude files or folders from being linted with Pylint.",
+    "settings.importStrategy.description": "Defines which Pylint to be used to lint Python files. When set to `useBundled`, the extension will use the Pylint binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the Pylint binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid Pylint binary in the selected environment, it will fallback to using the Pylint binary that is shipped with the extension The `pylint.path` setting may also be ignored when this setting is set to `fromEnvironment`.",
```

Comment:
```suggestion
    "settings.importStrategy.description": "Defines which Pylint binary to be used to lint Python files. When set to `useBundled`, the extension will use the Pylint binary that is shipped with the extension. When set to `fromEnvironment`, the extension will attempt to use the Pylint binary and all dependencies that are available in the currently selected environment. Note: If the extension can't find a valid Pylint binary in the selected environment, it will fallback to using the Pylint binary that is shipped with the extension The `pylint.path` setting may also be ignored when this setting is set to `fromEnvironment`.",
```